app/controllers/catalog_controller.rb

```diff
@@ -25,6 +25,7 @@ class CatalogController < ApplicationController
   end
 
   configure_blacklight do |config|
+    config.http_method = :post
```

Comment:
Any performance implications or other side effects we should be concerned about here?

Reply:
It makes debugging using Solr log files harder 🤸‍♂

Reply:
😢 seems like this should have been broken for year(s) already. I'm curious if thats worth the trade off or we have other alternatives? (batched requests? 😢 )